sky/utils/schemas.py

```diff
@@ -722,6 +722,14 @@ def get_config_schema():
         },
     }
 
+    allowed_clouds = {
+        # A list of cloud names that are allowed to be used
+        'type': 'array',
+        'items': {
+            'type': 'string',
```

Comment:
Do we want to make sure that the values are within the clouds supported?

```suggestion
            'type': 'string',
            'case_insensitive_enum': list(service_catalog.ALL_CLOUDS),
```